packages/color/src/Color.ts

```diff
@@ -207,6 +207,37 @@ export class Color
         return this._int;
     }
 
+    /**
+     * Convert to a hexadecimal number in little endian format (e.g., BBGGRR).
+     * @example
+     * import { Color } from 'pixi.js';
+     * new Color(0xffcc99).toLittleEndianNumber(); // returns 0x99ccff
+     * @returns {number} - The color as a number in little endian format.
+     */
+    toLittleEndianNumber(): number
+    {
+        const value = this._int;
+
+        return (value >> 16) + (value & 0xff00) + ((value & 0xff) << 16);
+    }
+
+    /**
+     * Multiply the colors
+     * @param {PIXI.Color | number[]} value - The color to multiply by, either
+     *   and existing Color object or an [R, G, B] or [R, G, B, A] array of floats.
+     */
+    multiply(value: Color | number[]): this
```

Comment:
Wouldn't it be better to accept a `ColorSource`?

Reply:
Yeah, you're probably right. I was trying to avoid `new Color` internally here, but I think it's probably okay.